test/e2e/testdata/eb-kafka-local.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: argoproj.io/v1alpha1
+kind: EventBus
+metadata:
+  name: test-eventbus
+  finalizers:
+  - eventbus-controller
+  generation: 2
+spec:
+  kafka:
+    exotic:
+      url: kafka1-0.kafka1.argo-events.svc.cluster.local:9092
```

Comment:
I'm wondering if we should address this on localhost for the functional tests. I imagine port forwarding would need to be set up, but I think the tests already do this for NATs/Jetstream eventbuses eh?